packages/stack/embarkjs/src/embarkjs-artifact.js.ejs

```diff
@@ -1,23 +1,35 @@
 import EmbarkJS from 'embarkjs';
```

Comment:
Since in Embark v5 we also have dapps specifying `embarkjs` in their `package.json` files, I think this `import` should be converted to a `require` and wrapped in try/catch (with helpful error message) like you've done for the `embarkjs-*` packages.

Reply:
I agree with this, but upon attempting to do this, it seems to be a bit more complex than it looks like and with a broader scope than the intention of this task, so will add this as a new task and shall be done in a different PR etc..

Reply:
https://www.pivotaltracker.com/story/show/170752860